packages/cli/src/services/config-service.ts

```diff
@@ -28,7 +28,18 @@ function readProjectConfig(userProjectPath: string): Promise<BoosterConfig> {
   return new Promise((resolve): void => {
     const projectBooster: BoosterApp = userProject.Booster
     projectBooster.configureCurrentEnv((config: BoosterConfig): void => {
+      checkEnvironmentWasConfigured(config)
       resolve(config)
     })
   })
 }
+
+function checkEnvironmentWasConfigured(config: BoosterConfig): void {
+  if (!config.configuredEnvironments.includes(config.environmentName)) {
+    const errorMessage = config.configuredEnvironments.length
+      ? `The environment '${config.environmentName}' does not match with any of the environments` +
+        ` you used to configure your Booster project which are: '${config.configuredEnvironments.join(', ')}'`
+      : "You haven't configured any environment. It is required to specify the deployment environments in 'src/config/config.ts'"
```

Comment:
Take into account that the file where the user puts its config can be anywhere and have any name (it is common to have several files, one per environment, like `config/production.ts` and `config/local.ts`). So I would change this message to something like:
```suggestion
      : "You haven't configured any environment. Please make sure you have at least one environment configured by calling `Booster.configure` method (normally done inside the folder 'src/config')"
```